src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx

```diff
@@ -55,20 +56,24 @@ export default class ProposalSummaryNFTManager extends React.Component<IProps, n
                 {decodedCallData.values[0]}
               </span>
             }
-            { detailView &&
+            {detailView &&
               <div className={css.summaryDetails}>
                 <div>
                   <strong>Send NFT </strong>
                   <img className={css.iconPadding} src="/assets/images/Icon/Transfer.svg" />
                   <a href={linkToEtherScan(decodedCallData.values[0])} target="_blank" rel="noopener noreferrer">{decodedCallData.values[0]}</a>
                 </div>
-                <br/>
+                <br />
                 <div>
                   <strong>NFT Contract:</strong> <a href={linkToEtherScan(decodedCallData.values[1])} target="_blank" rel="noopener noreferrer">{decodedCallData.values[1]}</a>
                 </div>
                 <div>
                   <strong>TokenID:</strong> {decodedCallData.values[2]}
                 </div>
+                <div>
+                  <strong>Raw call data:</strong>
```

Comment:
Don't we prefer to eliminate the use of `<strong>` and define the layout using div and CSS?

Reply:
I just didn't feel like refactoring the entire thing

Reply:
On the code that is coming in from outside contributors, we can hopefully get them to follow these standards before accepting their PRs.